activemerchant.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency("offsite_payments", "~> 2.0.0")
 
   s.add_development_dependency('rake')
+  s.add_development_dependency('test-unit', '~> 1.2.3')
```

Comment:
We don't support Ruby 1.8 anymore, right? Not saying that changes the decision, but maybe it means we can use a newer version of the `test-unit` gem?